holoviews/core/data/dask.py

```diff
@@ -229,12 +229,8 @@ def dframe(cls, columns, dimensions):
         return columns.data.compute()
 
     @classmethod
-    def length(cls, dataset):
-        """
-        Length of dask dataframe is unknown, always return 1
-        for performance, use shape to compute dataframe shape.
-        """
-        return 1
+    def nonzero(cls, dataset):
+        return True
 
```

Comment:
Discussed it properly now, and it's fairly clear that there won't be a cheap general solution here. A dask dataframe can be the result of a bunch of chained operations which all have to be evaluated before the length or even a nonzero length can be determined.